src/System.Net.Requests/ref/System.Net.Requests.cs

```diff
@@ -128,6 +128,7 @@ public partial class HttpWebRequest : System.Net.WebRequest
         internal HttpWebRequest() { }
         public string Accept { get { throw null; } set { } }
         public virtual bool AllowReadStreamBuffering { get { throw null; } set { } }
+        public string Connection { get { throw null; } set { } }
```

Comment:
>public string Connection { get { throw null; } set { } } [](start = 8, length = 56)

Why as part of this PR?  It's also being added in this PR:
[https://github.com/dotnet/corefx/pull/13441](https://github.com/dotnet/corefx/pull/13441)